src/main/java/net/neoforged/moddevgradle/internal/ModDevPlugin.java

```diff
@@ -234,13 +209,9 @@ public void apply(Project project) {
 
         // Configure common properties of NeoFormRuntimeEngineTask
         Consumer<NeoFormRuntimeTask> configureNfrtTask = task -> {
```

Comment:
This could be `tasks.getByType(...).configureEach` or similar at this point. Maybe we can even inline it since it's so trivial now.